GS-135/Add-additional comments

mananb77 opened the pull request:
### Jira Ticket

<!-- Replace the field marked <ticket-id> with your ticket id -->
[Jira Ticket](https://dashbd.atlassian.net/browse/<ticket-id>)

### Description

Add additional comments to the gradescope to sheets script such that it is easier to understand the functions.

### Type of Change

<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [x] Refactoring (non-breaking change)
- [] Breaking change (fix or feature that would change existing functionality)

### Changes

<!-- List the major changes made in this pull request. -->
Code Comments

### Testing

<!-- Describe how the feature has been tested, including both automated and manual testing strategies. -->
Manual testing with the python3 gradescope_to_sheets file.
Automated testing still to be COMPLETE.

### Checklist

- [x] My branch name matches the format: `<ticket-id>/<brief-description-of-change>`
- [x] My PR name matches the format: `[<ticket-id>] <brief-description-of-change>`
- [x] I have added doc-comments to all new functions ([JSDoc](https://jsdoc.app/) for JS and [Docstrings](https://peps.python.org/pep-0257/) for Python)
- [x] I have reviewed all of my code
- [x] My code only contains major changes related to my ticket

### Screenshots/Video

<!-- Include screenshots or video demonstrating the new feature, if applicable. -->

### Additional Notes

<!-- Any additional information or context relevant to this PR. -->

mananb77 committed:
updated line 84 and 85, now 94 and 95 due to the comments I had added in the code

mananb77 committed:
Tested the scripts and work with additional comments

mananb77 commented on gradescopeCronJob/config/cs10_fall2024.json:
```diff
@@ -1,7 +1,7 @@
 {
   "GRADESCOPE_COURSE_ID": "831412",
   "SCOPES": ["https://www.googleapis.com/auth/spreadsheets"],
-  "SPREADSHEET_ID": "1bqXOiAOYnEUD1FHoK1f_uX4PP3gplRR8maQr3S1zmTQ",
```
Should keep the original spreadsheet ID.

mananb77 replied:
Using another test id.

nawoda2 replied:
which spreadsheet id are we using

mananb77 commented on gradescopeCronJob/gradescope_to_spreadsheet.py:
```diff
@@ -35,7 +35,7 @@
     format="%(asctime)s - %(levelname)s - %(message)s",
     handlers=[
-        logging.FileHandler("/var/log/cron.log"),  # Logs to file
+        # logging.FileHandler("/var/log/cron.log"),  # Logs to file, comment out line for local testing ONLY
```
Need to uncomment.

mananb77 replied:
Uncommented.

mananb77 reviewed (commented):
There a few changes.

mananb77 committed:
tested code with script and cron job in docker container, works correctly

mananb77 committed:
script is tested and working

mananb77 commented on gradescopeCronJob/config/cs10_fall2024.json:
```diff
@@ -1,7 +1,7 @@
   "GRADESCOPE_COURSE_ID": "831412",
   "SCOPES": ["https://www.googleapis.com/auth/spreadsheets"],
-  "SPREADSHEET_ID": "1bqXOiAOYnEUD1FHoK1f_uX4PP3gplRR8maQr3S1zmTQ",
+  "SPREADSHEET_ID": "17ost05Bwy6pazdT7hDECxB670SQ0u3pTKA3dAIVTM7M",
```
Change to this: 10AH28_b0CQjJtf97DN3tY0-8P7Yaz-OTPmUh-V8SVC4

mananb77 committed:
Update cs10_fall2024.json

Created a new production sheet linked to this: 
https://docs.google.com/spreadsheets/d/10AH28_b0CQjJtf97DN3tY0-8P7Yaz-OTPmUh-V8SVC4/edit?gid=0#gid=0

nawoda2 reviewed (approved):
LGTM spreadsheet goes to Manan Fall 2024 Production Sheet

mananb77 reviewed (commented):
Resolved comments.
